propagation/toolboxcloud.md

```diff
@@ -1,10 +1,25 @@
-### Toolbox Cloudification Service
+### Toolbox Cloudification Service 
 
-The Toolbox Cloudification Service focuses on porting functionality from existing (open source) toolboxes into the EO platform
-targeted by your project. This service is designed to enhance the scalability and integration of your toolbox's features
-within an APEx compatible cloud environment.
+The toolbox cloudification service makes available single data processing (not visualization) functions of existing toolboxes on APEx. These data processing operations can then be freely combined with other algorithms hosted on APEx. Instead of a file-centric and local processing scheme of a traditional toolbox, key operations can be performed in the cloud where EO data archives are directly available as data cubes, where arbitrary extends in time and space can be extracted. Furthermore, functions from various toolboxes can be combined in a single workflow through APEx. 
 
-The process begins with an assessment of your request to estimate the effort required to cloudify the toolbox. The main
-advantage of this approach is that the specific functionality will also become available as a modular building block for
-other projects. Therefore, it is crucial to provide a compelling justification for why this feature deserves to be 'cloudified'.
+Service users may select functionalities from a toolbox that they would like to use on the APEx platform. The proposed functionalities and their toolbox will then be reviewed by ESA and APEx experts. Similar to the algorithm enhancement service, an effort estimation and an offer will be made following the review. On acceptance of the offer, the requested functionalities will be provided as an Application Package according to OGC best practice, or as an openEO User Defined Function or Process (UDF/UDP). 
```

Comment:
> Service users may select functionalities from a toolbox that they would like to use on the APEx platform.
I woud not use APEx platform as APEx does not provide any platform. It provides services that are compatible with existing NoR platforms, such as openEO Platform, CDSE, Ellip. 

Suggestion: Service users can propose capabilities from existing toolboxes to be extracted and transformed into APEx-compliant algorithms.

Reply:
> Similar to the algorithm enhancement service,

I would remove this part of the text as this assumes that the reader also read the algorithm enhancement page, which could not be the case.

Reply:
> or as an openEO User Defined Function or Process (UDF/UDP). 

This would be only as a UDP as this can easily be shared with others.